src/pages/[language]/[slug]/index.query.graphql

```diff
@@ -62,6 +63,35 @@ query LandingPage($locale: SiteLocale, $slug: String) {
           ...link
         }
       }
+      ...on SectionStructuredTextRecord {
+        body {
+          value
+          blocks {
+            __typename
+            ... on StructuredTextBlueTextRecord {
+              id
+              body {
+                value
+              }
+            }
+            ... on StructuredTextCtaListRecord {
+              id
+              ctas {
+                ...link
+              }
+            }
+            ... on StructuredTextHighlightedListRecord {
+              id
+              items {
+                body {
+                  value
+                }
+              }
+            }
+          }
+        }
+        gridAlignment
```

Comment:
Barely worth mentioning but if you agree let's put `gridAlignment` above `body`, tiny bit more readable I think.